docs/schemaValidationMiddleware.md

```diff
@@ -0,0 +1,126 @@
+# The Schema "Gatekeeper" Validation Middleware
+
+Although you usually will build some fault tolerance into your components, on some level they expect to deal with
```

Comment:
This apparently value-neutral use of terms like "right", "expected", "invalid", "correct" should be qualified in the first paragraph that you are only prepared to enter into this issue to the extent that the JSON schema standard can process.